packages/mocha-runner/src/mocha-runner.ts

```diff
@@ -137,10 +137,21 @@ class MochaRunner implements TestRunner {
             results.testResults = Util.filterTestResultsByTestLocator(results.testResults,
                 this._testlocator, this._blockTestLocators);
         }
-        mocha.dispose();
         // clearing blockTests and suites for next run
         this._blockTests = [];
         this._blockedSuites = [];
+        try {
+            mocha.dispose()
+        }
+        catch(err)
+        {
+            // implies user is using mocha version < 7.2
+            console.warn("Using mocha < 7.2", err);
```

Comment:
remove these [at least the err stacktrace]. We're planning to remove err stacktrace at the other place too as it is confusing to the end developer

Reply:
removed